spec.bs

```diff
@@ -1103,11 +1105,15 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|:
   1. Otherwise, if |winner|'s [=generated bid/modified bid=] is not null, [=map/set=]
     |browserSignals|["`modifiedBid`"] to it.
   1. Set |config| to |leadingBidInfo|'s [=leading bid info/auction config=].
+  1. Set |sellerSignals| to |config|'s [=auction config/seller signals=].
+  1. If |sellerSignals| is a {{Promise}}:
+    1. [=Upon fulfillment=] of |sellerSignals| with |result| set |config|'s

```

Comment:
I'm just realizing that it looks like this is all done in parallel, right? We can't wait for promise fulfillment in parallel, just as we can't send a v8::Promise or blink::ScriptPromiseResolver to the browser process etc and tap into the Blink/v8 event loop.

Instead, we have to basically mirror [the implementation here](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:third_party/blink/renderer/modules/ad_auction/navigator_auction.cc;l=2249-2267;drc=a129ef22074b9f81f549ff068a15fc320072b3bb), which waits for the promise to resolve in the renderer, and messages the browser about it asynchronously so it can continue with whatever seller information it needs as it comes in.

Reply:
Okay I think I fixed it all up.